packages/SwingSet/src/lib/message.js

```diff
@@ -72,7 +72,7 @@ export function insistVatDeliveryObject(vdo) {
       break;
     }
     case 'stopVat': {
-      assert(rest.length === 0);
+      assert(rest.length === 1);
```

Comment:
This wants extra asserts for the properties of the new argument, like how `case 'message'` and `case 'notify'` do it.